src/lib/viewers/BaseViewer.js

```diff
@@ -1000,6 +1007,35 @@ class BaseViewer extends EventEmitter {
         return AnnotationMode.NONE;
     }
 
+    /**
+     * Sets the cursor ftux key in localStorage
+     *
+     * @protected
+     * @param {string} key
+     * @return {void}
+     */
+    setCursorFtux(key) {
```

Comment:
We want to reduce the amount of code we're introducing to BaseViewer. This method seems to have very little code in it, so its logic can be moved to the subclasses.